content/part3.md

```diff
@@ -0,0 +1,303 @@
+# Building an eCommerce web app with Temporal, Part 3: Testing
+
+In [Part 1](https://github.com/temporalio/temporal-ecommerce/blob/main/content/part1.md) and [Part 2](https://github.com/temporalio/temporal-ecommerce/blob/main/content/part2.md), you built out a shopping cart with an abandoned cart email notification using long-lived Workflows.
+Using Workflows and Activities, you can easily build features that would be tricky in a traditional RESTful API, like sending an email reminder when a user hasn't touched their cart in a while.
+Temporal Workflows do more than make working with time easy, they also make it easier to test Workflows that depend on time.
+In this blog post, I'll demonstrate how to use Temporal's testing utilities to write fast unit tests for the shopping cart from Part 1 and Part 2.
```

Comment:
some possible extra content (take it or leave it)

- temporal handles some of your most business critical code, you should want a proper testing solution to avoid regressions
- descript case study explicitly calls out the value of testing https://docs.temporal.io/blog/descript-case-study